src/actions/firestore.js

```diff
@@ -97,6 +97,7 @@ export function get(firebase, dispatch, queryOption) {
         payload: (snap) => ({
           data: dataByIdSnapshot(snap),
           ordered: orderedFromSnap(snap),
+          fromCache: (snap.metadata && snap.metadata.fromCache) || true,
```

Comment:
Isn't it always resolved to `true`?

With `const snap = { metadata: { fromCache: false } }`:

```
false || true -> true
```

With `const snap = { metadata: { fromCache: true } }`:

```
true || true -> true
```

Reply:
Good catch. I need to do a boolean check instead.

Reply:
fixed.